internal/k8s/controller_test.go

```diff
@@ -159,9 +159,20 @@ func TestIsNginxIngress(t *testing.T) {
 func TestCreateMergableIngresses(t *testing.T) {
 	cafeMaster, coffeeMinion, teaMinion, lbc := getMergableDefaults()
 
-	lbc.ingressLister.Add(&cafeMaster)
-	lbc.ingressLister.Add(&coffeeMinion)
-	lbc.ingressLister.Add(&teaMinion)
+	err := lbc.ingressLister.Add(&cafeMaster)
+	if err != nil {
+		t.Errorf("Error adding Ingress to the ingress lister: %v", err)
```

Comment:
can we include the name of the Ingress here and in other places - `"Error adding Ingress %v to the ingress lister: %v", cafeMaster.Name, err` ?

Reply:
Ok